syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -56,6 +56,7 @@ class Syft private constructor(
                 }
                 .subscribeOn(schedulers.computeThreadScheduler)
                 .observeOn(schedulers.calleeThreadScheduler)
+                .onErrorReturn{ print("Something went wrong $it") }
```

Comment:
this is removed in the new pattern ;p

Reply:
We can remove it after merging the other PR. This line fixes a crash.